test/gst-va/encode/10bit/vp9.py

```diff
@@ -0,0 +1,94 @@
+###
+### Copyright (C) 2023 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+from .....lib import *
+from .....lib.gstreamer.va.util import *
+from .....lib.gstreamer.va.encoder import EncoderTest
+
+spec = load_test_spec("vp9", "encode", "10bit")
+
+@slash.requires(*have_gst_element("vavp9lpenc"))
+@slash.requires(*have_gst_element("vavp9dec"))
+class VP9_10EncoderBaseTest(EncoderTest):
+  def before(self):
+    super().before()
+    vars(self).update(
+      codec         = "vp9",
+      gstencoder    = "vavp9lpenc",
```

Comment:
This should be in the `VP9_10EncoderLPTest` subclass

Reply:
updated

Reply:
Hmm.. not updated

Reply:
Updated,
I missed understand last time, how about now?